lit_gpt/config.py

```diff
@@ -781,6 +781,49 @@ def norm_class(self) -> Type:
         configs.append(copy)
 
 
+###############
+# Google Gemma
+###############
+gemma = [
+    # https://huggingface.co/google/gemma-7b/blob/main/config.json
+    dict(
+        name="Gemma-7b-hf",
+        hf_config=dict(org="google", name="gemma-7b"),
+        vocab_size=256000,
+        padding_multiple=64,
+        n_embd=3072,
+        n_layer=28,
+        n_head=16,
+        n_query_groups=1,
```

Comment:
The report says that 7b uses MHA (the default)

```suggestion
```

Reply:
I moved 2b at the top so now these comments might seem confusing.